### PR TITLE
README: Better document cursor reuse

### PR DESCRIPTION
**Pull request by jrodewig**
### Description
 Documents for feeds that:

* Event replay only occurs if you pass an event source to `feed()`
* If you pass an event source query to `feed()`, the event source is created at the same time as the request.
* You can reuse cursors across event sources with identical queries in the same database. 

### Motivation and context
Adds related docs for https://github.com/fauna/fauna-js/pull/298.

### How was the change tested?
N/A

### Change types

* - [ ] Bug fix (non-breaking change that fixes an issue)
* - [ ] New feature (non-breaking change that adds functionality)
* - [ ] Breaking change (backwards-incompatible fix or feature)

### Checklist:

* - [ ] My code follows the code style of this project.
* - [x] My change requires a change to Fauna documentation.
* - [x] My change requires a change to the README, and I have updated it accordingly.

----
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -592,6 +598,10 @@ const options: FeedClientConfiguration = {
 client.feed(fql`Product.all().eventSource()`, options);
 ```
 
+You can reuse
```
Comment (jrodewig) [inline README.md:601]: @erickpintor Can you confirm this? Also let me know if there are any potential pitfalls to point out here. Thanks!

Comment (erickpintor) [reply README.md:601]: That's true. Cursors are meant to be track your position wrt. the event feed you're consuming. The event source shape is deterministic, though. You can either use the same event source you already have or regenerate it via the same FQL query.

**Review by erickpintor — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (jrodewig) [reply README.md:601]: Awesome. Thank you!

**Review by ecooper — APPROVED** [review]
Thanks for doing this update!

**Pull request merged**